.github/workflows/welcome-newcomers.yaml

```diff
@@ -0,0 +1,40 @@
+name: first-interaction
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    branches: [main]
+    types: [opened]
+
+jobs:
+  check_for_first_interaction:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Welcome to the Pantsbuild Community. This looks like your first issue here.
+
+            If you haven't already, come say hi on Slack.
+            Use [this link](https://docs.google.com/forms/d/e/1FAIpQLSf9zgf-MXRnVDJbrVEST3urqneq7LCcy0zw8qU-GH4hPMn52A/viewform?usp=sf_link)
```

Comment:
If you wanted to, the text here and below could be tightened like `... come [say hi on Slack](https://...)` and save this extra sentence.